cfgmgr/coppmgr.cpp

```diff
@@ -474,8 +548,54 @@ void CoppMgr::doCoppTrapTask(Consumer &consumer)
                 (trap_group == m_coppTrapConfMap[key].trap_group) &&
                 (trap_ids == m_coppTrapConfMap[key].trap_ids))
             {
-                it = consumer.m_toSync.erase(it);
-                continue;
+                std::vector<std::string> feature_keys;
```

Comment:
Please don't add code inside duplicate check. The duplicate check is meant to return without processing when config is duplicate.
Let's assume if we have ability to change two fields in future(e.g. trap_group and always_enabled). This logic will not work.

Reply:
Revisit the duplicate check and modify it according to new field added.

Reply:
Done